lua/wikis/commons/Widget/Match/Card.lua

```diff
@@ -0,0 +1,125 @@
+local MatchPageButton = Lua.import('Module:Widget/Match/PageButton')
+local Button = Lua.import('Module:Widget/Basic/Button')
+local ImageIcon = Lua.import('Module:Widget/Image/Icon/Image')
+local StreamsContainer = Lua.import('Module:Widget/Match/StreamsContainer')
+
+local HIGHLIGHT_CLASS = 'tournament-highlighted-bg'
+local SHOW_STREAMS_WHEN_LESS_THAN_TO_LIVE = 2 * 60 * 60 -- 2 hours in seconds
+
+---@class MatchCard: Widget
+---@operator call(table): MatchCard
+local MatchCard = Class.new(Widget)
+MatchCard.defaultProps = {
+	hideTournament = false, -- Hide the tournament and stage
+	displayGameIcons = false, -- Display the game icon in the tournament title
+	onlyHighlightOnValue = nil, -- Only highlight if the publishertier has this value
+}
+
+---@return Widget?
+function MatchCard:render()
+	---@type MatchGroupUtilMatch
+	local match = self.props.match
+	if not match then
+		return nil
+	end
+
+	local matchPhase = MatchGroupUtil.computeMatchPhase(match)
+
+	local displayMatchPage = Info.config.match2.matchPage and matchPhase ~= 'upcoming'
+	local displayVods = matchPhase == 'finished'
+	local displayStreams = matchPhase == 'ongoing'
+	-- Show streams also for the last period before going live
+	if matchPhase == 'upcoming' and
+		os.difftime(match.timestamp, DateExt.getCurrentTimestamp()) < SHOW_STREAMS_WHEN_LESS_THAN_TO_LIVE then
+
+		displayStreams = true
+	end
+
+	local highlightCondition = HighlightConditions.match or HighlightConditions.tournament
+	local highlight = highlightCondition(match, {onlyHighlightOnValue = self.props.onlyHighlightOnValue})
+
+	---@param vod string?
+	---@param index integer?
+	---@param callToAction boolean
+	---@return Widget?
+	local makeVodButton = function(vod, index, callToAction)
+		if Logic.isEmpty(vod) then
+			return nil
+		end
+		---@cast vod -nil
+		return Button{
+			linktype = 'external',
+			title = VodLink.getTitle(index),
+			variant = 'tertiary',
+			link = vod,
+			size = 'sm',
+			grow = callToAction,
+			children = {
+				ImageIcon{imageLight = VodLink.getIcon(index)},
+				callToAction and ' ' or nil,
+				callToAction and VodLink.getTitle(index) or nil,
+			},
+		}
+	end
+
+	local tournamentLink = TournamentTitle{
+		tournament = Tournament.partialTournamentFromMatch(match),
+		displayGameIcon = self.props.displayGameIcons,
+		stageName = match.section,
+	}
+
+	local matchPageButton = MatchPageButton{
+		matchId = match.matchId,
+		hasMatchPage = Logic.isNotEmpty(match.bracketData.matchPage),
+	}
+
+	return HtmlWidgets.Div{
+		classes = {'match-info', highlight and HIGHLIGHT_CLASS or nil},
+		children = WidgetUtil.collect(
+			MatchCountdown{match = match},
+			MatchHeader{match = match},
+			not self.props.hideTournament and HtmlWidgets.Div{
+				classes = {'match-info-tournament'},
+				children = {tournamentLink},
+			} or nil,
+			HtmlWidgets.Div{
+				classes = {'match-info-links'},
+				children = WidgetUtil.collect(
+					displayMatchPage and matchPageButton or nil,
```

Comment:
oos, but imo for wikis that do not have match pages we should link to a prefilled form which just displays the match as single match
(would need 1 small form bot created on all wikis buls simple link building for the prefilled form here)